test/presenters/document_collection_presenter_test.rb

```diff
@@ -124,4 +124,21 @@ class GroupWithDocumentsWhenThereIsNoLinksDocuments < TestCase
       assert_empty group_with_missing_documents
     end
   end
+
+  class GroupWithDocumentsWhenThereAreWithdrawnDocuments < TestCase
+    test "does not present withdrawn documents" do
+      presenter = presented_item(
+        "document_collection_with_withdrawn_links_documents"
+      )
+
+      groups_without_withdrawn_documents = presenter.groups.first["documents"]
```

Comment:
I don't think this name correctly describes what is returned here. This returns (or should return for the test to pass) a collection of non-withdrawn documents. Maybe `non_withdrawn_documents` might be a better name?